.github/workflows/wheel-builder.yml

```diff
@@ -119,7 +119,8 @@ jobs:
           - {VERSION: "3.6", MSVC_VERSION: "2019"}
           - {VERSION: "3.7", MSVC_VERSION: "2019"}
           - {VERSION: "3.8", MSVC_VERSION: "2019"}
-          - {VERSION: "3.8", MSVC_VERSION: "2019", "USE_ABI3": "true", "ABI_VERSION": "cp36"}
+          - {VERSION: "3.9", MSVC_VERSION: "2019"}
```

Comment:
I’d be fine not generating this wheel. If you’re on 3.9 you should have an abi3 aware pip.

Reply:
Works for me! I'm pro abi3 only.